packages/fxa-auth-server/lib/routes/recovery-codes.js

```diff
@@ -54,6 +60,13 @@ module.exports = (log, db, config, customs, mailer, glean) => {
           RECOVERY_CODE_COUNT
         );
 
+        accountEventsManager.recordSecurityEvent(db, {
```

Comment:
I'm presuming that you are not awaiting by choice.  But just in case: you are not awaiting. :)

Reply:
Yea, didn't want to block the request on storing this data.